src/App.php

```diff
@@ -1193,4 +1224,82 @@ private function getRenderedPortals(): array
 
         return $portals;
     }
+
+    /**
+     * Return true if $_GET[$key] exists.
+     */
+    public function issetRequestGetParam(string $key): bool
+    {
+        return isset($this->request->getQueryParams()[$key]);
+    }
+
+    /**
+     * Return whole $_GET array data.
+     */
+    public function getRequestGetParams(): array
+    {
+        return $this->request->getQueryParams();
+    }
+
+    /**
+     * Return $_GET param by key or null if not exists.
+     *
+     * @return mixed
+     */
+    public function getRequestGetParam(string $key)
+    {
+        return $this->getRequestGetParams()[$key] ?? null;
```

Comment:
with https://github.com/atk4/ui/pull/2101/files#r1316302660 returning `true` for `null`, this should throw if the key is not present